web/discord_relay/queue.go

```diff
@@ -72,8 +118,18 @@ func queueGroomer() {
 			return
 		}
 
-		log.Print(message.SteamName, message.SteamId, message.Content)
+		log.Print(message.Type, message.SteamName, message.SteamId, message.Content)
```

Comment:
```suggestion
		log.Println(message.Type, message.SteamName, message.SteamId, message.Content)
```
i think this should be Println. idk why it was print before, perhaps @brandonsturgeon could comment.